src/bramble/client/main.js

```diff
@@ -136,8 +137,11 @@ define([
         _instance = new BrambleProxy(div, options);
     };
 
+//my code 
+//var projectLimiter = new ProjectLimiter();
+
     // After calling Bramble.load(), Bramble.mount() specifies project root entry path info
-    Bramble.mount = function(root, filename) {
+    Bramble.mount = function(root, filename,projectLimits) {
```

Comment:
projectLimits here is parameter that is passed from hosted.js that contains info about the maximum allowed project limits.